File: packages/shared/src/types/banner.ts
export interface BannerItem {
  id: string;
  title: string;
  subtitle: string | null;
  imageUrl: string;
  linkType: string;
  linkValue: string | null;
  sortOrder: number;
  isActive: boolean;
  startAt: string | null;
  endAt: string | null;
  createdAt: string;
  updatedAt: string;
}

File: apps/ios/Shop/Feature/Home/Data/Banner.swift
import Foundation

struct Banner: Codable, Equatable, Identifiable, Sendable {
    let id: String
    let title: String
    let subtitle: String?
    let imageUrl: String
    let linkType: String
    let linkValue: String?
    let sortOrder: Int
    let isActive: Bool
    let startAt: String?
    let endAt: String?
    let createdAt: String
    let updatedAt: String
}
